tasks/interactive-bots.md

```diff
@@ -94,14 +92,14 @@ by:
 * Take screenshots showing that the bot is working, make sure to have screenshots
 of:
 
- - your terminal window with the bot running, including the command you used to
+ - the output in your terminal window with the bot running, including the command you used to
  run the bot and the output with the bot description
- - messages sent to the bot in the `devel` and `social` streams
- - messages sent by the bot in the followup stream
+ - messages sent by you to the bot in the `devel` and `social` streams
+ - messages sent by the bot in the `followup` stream
  - any other screenshots you find relevant
 
 Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
-filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+filenames do not have spaces. Instead, use dashes (`-`).
```

Comment:
It's unclear to me if this clarifies anything, especially since "white space" is not the same as "space" - the former is a more general term used often when developing. I don't think this one is necessary.